app/components/channel_drawer/channels_list/channel_item/index.js

```diff
@@ -29,13 +29,17 @@ function makeMapStateToProps() {
             }
         }
 
+        const isReadOnly = isChannelReadOnlyById(state, channel.id);
+        const shouldHideChannel = !ownProps.isSearchResult && !ownProps.isFavorite && isReadOnly;
```

Comment:
@jasonblais You said previously that we could just hide the town square when it's read only since Uber are the main users of that setting, but we're using it on pre-release now, so we're going to be hiding the Reception channel if it fits these conditions. Is that still okay with us using it?

Reply:
Actually, I just realized that we already do this on the web app

Reply:
Yeah, can just be consistent with the web app in this case.